src/deployments/runtime/src/ou-validation-events/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ScheduledEvent } from 'aws-lambda';
+export function getInvoker(input: ScheduledEvent): string | undefined {
+  const requestDetail = input.detail;
+  if (
+    requestDetail &&
+    requestDetail.userIdentity &&
+    requestDetail.userIdentity.sessionContext &&
+    requestDetail.userIdentity.sessionContext.sessionIssuer
+  ) {
```

Comment:
Shouldn't this be something like the following:

if (requestDetail &&
    requestDetail.userIdentity &&
    requestDetail.userIdentity.sessionContext &&
    requestDetail.userIdentity.sessionContext.sessionIssuer) {
.....
.....
} else if (
requestDetail &&
requestDetail.userIdentity &&
requestDetail.userIdentity.username
) {
return requestDetail.userIdentity.userName; //shouldn't this be returned?
}

Reply:
We just want to ignore if action is performed by PBMMAccel-SM-Execution-Role and that comes by AssumeRole. That is why only checking for that role name and ignoring if it is invoced by SM-Execution-Role. Other that that we will perform necessary action.